test/sequential/test-inspector-ip-detection.js

```diff
@@ -30,7 +30,7 @@ function pickIPv4Address() {
 }
 
 async function test() {
-  const instance = new NodeInstance('--inspect-brk=0.0.0.0:0');
+  const instance = new NodeInstance('--inspect-brk=[::]:0');
```

Comment:
Keep IPv4 here for hosts that do not support IPv6.
```suggestion
  const instance = new NodeInstance('--inspect-brk=0.0.0.0:0');
```